### ISSUE TITLE
[Question] Does it make sense to use this app outside of China?

### ISSUE THREAD

**Issue by 000exploit** [gmg137/netease-cloud-music-gtk#165]
Hello, I saw this project just a few minutes ago in Flathub, and I was very interested in it. There's beautiful design, and it looks like a great music streaming service! Unfortunately, it's not adapted for phones, but it's still good. So, the question is, can I use this app for listening music in _Europe_, and is it possible? And no, a very different style of music does not bother me in any way.

**Comment by gmg137** [comment]
Official apps are recommended on mobile phones: https://music.163.com

**Comment by 000exploit** [comment]
> Official apps are recommended on mobile phones: https://music.163.com

I mean mobile phones like PinePhone (Pro) or Librem 5, where linux is the only option.

**Comment by gmg137** [comment]
I need to study whether these mobile phone systems can use GTK for application development.

**Comment by Henry-Hiles** [comment]
> I need to study whether these mobile phone systems can use GTK for application development.

Yes. Your app already runs, it just doesn't fit well, you need to make things wrap and such so it fits on the screen.
